src/pyfaf/opsys/centos.py.in

```diff
@@ -32,7 +32,9 @@
                            ReportPackage,
                            ReportUnknownPackage,
                            column_len)
-from pyfaf.repos.yum import Yum
+# autotools replaces @sw_mgr@ with either yum or dnf
```

Comment:
I just remembered that there is `repo_types` variable. `from pyfaf.repos import repo_types`. It can be easily used, instead of changing files.

```
from pyfaf.repos import repo_types
prefered_repos = ["dnf", "yum", "rpmmetadata"]
for repo_type in prefered_repos:
    if repo_type in repo_types:
        print(systems[repo_type].list_packages(["src"]))
        break
else:
    print("No repo available")
```
I believe this approach can be used on all places where the substitution is used.